### DISCUSSION TITLE
Issue with postMessage: Failed to execute 'postMessage' on 'DOMWindow'

### DISCUSSION THREAD

**Discussion by ruben-certhis** [WalletConnect/walletconnect-monorepo#2653] [Developer Support]
hello i try to implement the v2 but i got the error :

index.es.js:82 Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://verify.walletconnect.com') does not match the recipient window's origin ('https://staging.certhis.io').

**Comment by glitch-txs** [comment]
hey are you still facing this? could you share a minimal reproducible example repo?
